src/main/groovy/com/netgrif/application/engine/petrinet/domain/dataset/UserListFieldValue.groovy

```diff
@@ -1,23 +1,32 @@
-package com.netgrif.application.engine.petrinet.domain.dataset;
+package com.netgrif.application.engine.petrinet.domain.dataset
 
 class UserListFieldValue {
 
-    private List<UserFieldValue> userValues;
+    private Set<UserFieldValue> userValues
 
-    UserListFieldValue(List<UserFieldValue> userValues) {
-        this.userValues = userValues;
+    UserListFieldValue() {
+        this.userValues = new LinkedHashSet<>()
     }
 
-    List<UserFieldValue> getUserValues() {
-        return userValues;
+    UserListFieldValue(Collection<UserFieldValue> userValues) {
+        this()
+        this.userValues = new LinkedHashSet<>(userValues)
+    }
+
+    LinkedHashSet<UserFieldValue> getUserValues() {
+        return userValues
+    }
+
+    void setUserValues(Set<UserFieldValue> userValues) {
```

Comment:
This could be simplified with `Collection` too

Reply:
Corrected.